modules/wkt/src/lib/encode-wkt.ts

```diff
@@ -38,24 +40,24 @@ export default function encodeWKT(gj: {
   }
 
   function wrapParens(s: string) {
-    return '(' + s + ')';
+    return `(${s})`;
   }
 
   switch (gj.type) {
     case 'Point':
-      return 'POINT (' + pairWKT(gj.coordinates as any[]) + ')';
+      return `POINT (${pairWKT(gj.coordinates as number[])})`;
     case 'LineString':
-      return 'LINESTRING (' + ringWKT(gj.coordinates as any[]) + ')';
+      return `LINESTRING (${ringWKT(gj.coordinates as number[][])})`;
     case 'Polygon':
-      return 'POLYGON (' + ringsWKT(gj.coordinates as any[]) + ')';
+      return `POLYGON (${ringsWKT(gj.coordinates as number[][][])})`;
     case 'MultiPoint':
-      return 'MULTIPOINT (' + ringWKT(gj.coordinates as any[]) + ')';
+      return `MULTIPOINT (${ringWKT(gj.coordinates as number[][])})`;
     case 'MultiPolygon':
-      return 'MULTIPOLYGON (' + multiRingsWKT(gj.coordinates as any[]) + ')';
+      return `MULTIPOLYGON (${multiRingsWKT(gj.coordinates as number[][][][])})`;
     case 'MultiLineString':
-      return 'MULTILINESTRING (' + ringsWKT(gj.coordinates as any[]) + ')';
+      return `MULTILINESTRING (${ringsWKT(gj.coordinates as number[][][])})`;
     case 'GeometryCollection':
-      return 'GEOMETRYCOLLECTION (' + gj.geometries.map(encodeWKT).join(', ') + ')';
+      return `GEOMETRYCOLLECTION (${gj.geometries!.map(encodeWKT).join(', ')})`;
```

Comment:
If you use the `geometry` type from `@types/geojson`, I don't think you'd need any of these casts.

Reply:
Another benefit to using the `Geometry` type is that you can assert `: never` after this last case to make it clear that we're doing all exhaustive checks for a `Geometry` object